Kitodo/src/main/java/org/kitodo/production/helper/VariableReplacer.java

```diff
@@ -102,11 +102,11 @@ public String replace(String inString) {
         // replace paths and files
         try {
             // TIFF writer scripts will have a path without an end slash
-            String processPath = replaceSlashAndSeparator(processService.getProcessDataDirectory(this.process));
```

Comment:
Is there some special reason to change order here?

Reply:
Yes, it was a checkstyle problem "assignment and usage are too fare away"